docs/faq.rst

```diff
@@ -693,6 +693,85 @@ Additional reading:
   * `How to avoid NaN gradients when using where <https://github.com/tensorflow/probability/blob/master/discussion/where-nan.pdf>`_.
 
 
+Gradients are zero for rank-based functions
```

Comment:
I have to admit it bothers me that our FAQ doesn't contain actual questions :P But maybe that's for a different PR to resolve.

Reply:
Fixed, at least for this case!